APIcall/data_collection.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import argparse
+from datetime import date
+
+def api_311(start_date, end_date, skip, limit):
+    skip = 0
+    limit = 10000

```

Comment:
please create a module-level constant for this, e.g., `REQUESTS_BATCH_SIZE = 10000`. You can put it under the imports.